action.yaml

```diff
@@ -48,13 +48,22 @@ runs:
         # This will set an environment variable we can use to conditionally
         # install pre-commit if needed and toggle how the action runs.
         echo "PRE_COMMIT_BIN=$(command -v pre-commit)" >> $GITHUB_ENV
+    - name: Find poetry
```

Comment:
I think it would be nice to add some comments here about why this is needed / what this is trying to fix

Reply:
I'll add more context, this is mainly coming from https://github.com/turo/orion/pull/457 where the poetry hooks seem to default to the default python version and not the one of the current environment.